feature-libs/asm/styles/components/_customer-list.component.scss

```diff
@@ -67,6 +67,10 @@ $sapFioriContent_LabelColor: #6a6d70;
             background-color: #ffffff;
           }
         }
+        .form-control::-webkit-input-placeholder {
```

Comment:
does this style works for every version?

Reply:
Done